Update Documentation with Pydata Sphinx Theme, and more (#523)
rapidsai/cuspatial: docs/source/api_docs/spatial_indexing.rst

Spatial Indexing
----------------

> Spatial indexing functions provide blisteringly-fast on-GPU point-in-polygon
> operations.

Reviewer: ```suggestion
Spatial indexing functions provide GPU-accelerated point-in-polygon
and spatial join operations.
```
